cumulusci/tasks/salesforce/UpdateDependencies.py

```diff
@@ -282,3 +282,43 @@ def _uninstall_dependency(self, dependency):
             self, package_zip(), purge_on_delete=self.options["purge_on_delete"]
         )
         return api()
+
+    def freeze(self, step):
+        dependencies = self.project_config.get_static_dependencies(
+            self.options["dependencies"], include_beta=self.options["include_beta"]
+        )
+        steps = []
+        for i, dependency in enumerate(_flatten(dependencies), start=1):
+            if "namespace" in dependency:
+                kind = "managed"
+                # @@@ we want the package name, not namespace
+                name = "Install {}".format(dependency["namespace"])
+            else:
+                kind = "metadata"
+                # @@@ we want a friendly name from...where?
```

Comment:
Good point.  We don't really have a place for that at the moment so the generated string is fine for now